### ISSUE TITLE
Build boards of UX and Visual inspiration (Landscaping)

### ISSUE THREAD

**Issue by thejoefriel** [yalla-coop/universal-credit-support#3]
**PLEASE NOTE, LINKS TO BOARDS TO THEN BE PASTED HERE:**
TBC

---

## Summary
In this issue we are trying to get a better sense of the type of user experience we want to try and build for our users (i.e. the best way for them to interact with our digital tool in order to achieve their needs), how it might look and the overall content style. 

**What other people are doing in this space:**
- What already exists and why it's not fitting the problems we're looking to solve - what are our users needing
- The language other platforms use - what works and what doesn't

**UX / User Experience:**
- User experiences our user personas know and like
- inspiration from platforms that we can use for how users might interact with our own digital platform
- what apps / websites do your clients tend to use?
- what apps / websites do your therapists use?

**Content:**
- What is the tone of the platforms and tools our personas use?
- What is the tone of other similar platforms looking to support people?

**Visual:**
- What visual language do you think suits the user personas in terms of typography, colour and graphics/illustrations. What would be attractive to them?
- What is the visual identity or the app? Are there Chiltern Music Therapy brand guidelines for us to work under?

**Accessibility**
This has an important part in our design. We need to agree what level of accessibility, following agreed standards. Our assumption would be AA as a minimum but aiming for AAA. 
Details here: https://www.w3.org/WAI/WCAG2AAA-Conformance

---

### Please watch below videos and check Acceptance Criteria for further instructions of how to do this exercise

**Please try and comment on this issue with your feedback by X**
- [ ] Using the template below, add comments on this issue for each digital tool you feel worth discussing

**Template:**
- App/website name:
- App/website link:
- What is it relevant for: Other Person in Space / UX / Visual / Content
- If possible, the specific part(s) and/or flow(s) you like (with link or a few screenshots if possible!) and a description of why you and our users would like it. Why do you feel it's useful to look at with our project in mind?

- [ ] List links / documents for any brand guidelines we need to follow

- [ ] Please let us know in a comment your views on the level of accessibility 

### Video explanations
https://www.loom.com/share/bce88cc1ee3f4aa9833fd8bdd439e94d
https://www.loom.com/share/4a8ee418f94e48b6a6435bd79420a3ac

**Comment by jema28** [comment]
Hey @maggiehoughton and @Katie-Hyde, I hope you're both well! 

I'm looking to work on preparing the landscaping session tomorrow morning before our meeting on Wednesday morning. Would you mind looking through the above and adding websites/app names that you like and an explainer of why you've added them?

Would you be able to do this by this late afternoon or early tomorrow morning? 

Thanks! Please let us know if you have any questions :)

**Comment by maggiehoughton** [comment]
[Hyde_Brand_Guidelines_V3p2_LowRes_(72dpi).pdf](https://github.com/yalla-coop/universal-credit-support/files/6803478/Hyde_Brand_Guidelines_V3p2_LowRes_.72dpi.pdf)
Brand Guidelines (finally worked out how to attach a file...)

**Comment by maggiehoughton** [comment]
•	App/website name: Stepchange 'Get help now'
•	App/website link: https://www.stepchange.org/start.aspx  
•	What is it relevant for:  flow, & visual
•	It's a bit of everything, but very friendly and easy to navigate

**Comment by maggiehoughton** [comment]
•	App/website name: Incomemax
•	App/website link: https://www.incomemax.org.uk/for-customers/
•	What is it relevant for:  visual & copy
They make it sound very easy!

**Comment by maggiehoughton** [comment]
Note on the above- there is an absolute void in the space we're working in re support for UC claimants (if you discount the content heavy resources provided by DWP and CAB etc- so these are just pages that are sort of related and have similar target groups, but do not necessarily deal with the exact issue at hand.

**Comment by maggiehoughton** [comment]
•	App/website name: NDL budgeting tool
•	App/website link: https://tools.nationaldebtline.org/yourbudget/
•	What is it relevant for:  visual + flow
It’s content heavy but I like that you can see the full process when you start so you always know how far you have to go.

**Comment by maggiehoughton** [comment]
•	App/website name: Payplan online debt advice
•	App/website link: https://www.payplan.com/free-online-debt-help/
•	What is it relevant for:  flow
I like the progress bar along the top, and also how it offers to direct you to further help when  needed

**Comment by maggiehoughton** [comment]
•	App/website name: Turn 2 us UC guide
•	App/website link: https://www.turn2us.org.uk/Benefit-guides/Universal-Credit-How-to-Claim/Getting-ready-to-claim
•	What is it relevant for: Other Person in Space / Content
•	The content is managed really well here, short and to the point.

**Comment by thejoefriel** [comment]
Hey @maggiehoughton @Katie-Hyde thanks for sending these over! Do you also have the content guide you mentioned? I.e. words to avoid

**Comment by maggiehoughton** [comment]
TLDR: avoid technical jargon (eg. use 'money' in stead of 'financial'), avoid the word 'debt', use I/you/we and keep it friendly.

https://asauk.org.uk/wp-content/uploads/2018/03/MAS0032-MAS-BehaviouralChange_W.pdf
https://www.hyde-housing.co.uk/media/1113/better-money-behaviours-a-toolkit.pdf

**Comment by Katie-Hyde** [comment]
not a advice web site but
• App/website name: Octopus energy
• App/website link: https://octopus.energy/
• What is it relevant for: easy of use
• The view is simple, easy language, know where to go, easy to navigate

**Comment by jema28** [comment]
Hey @Katie-Hyde thanks for the above! Do you have a screenshot you can share? I don't have an account with them, I can only see the main website

**Comment by Katie-Hyde** [comment]
https://octopus.energy/dashboard/new/#/accounts/A-7A6804D7/dashboard

hope that works for you Jem, if not I can share screens later

Katie Ford
Senior Advisor Successful Tenancies


T: 01243 816943 (Ext. 16943)
M: 07918 768 587
E: ***@***.******@***.***>

Hyde Foundation • Martlet House • Southern Gate • Chichester • PO19 8SG
www.hyde-housing.co.uk<http://www.hyde-housing.co.uk/>

Follow us: Twitter<https://twitter.com/HydeHousing> • Facebook<http://www.facebook.com/HydeGroup> • LinkedIn<http://www.linkedin.com/company/the-hyde-group> • YouTube<http://www.youtube.com/user/TheHydeGroup?feature=watch>


MyAccount - manage your account online anywhere, anytime<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.hyde-housing.co.uk%2Ftenants%2Fmyaccount%2F&data=04%7C01%7Ckatie.ford%40hyde-housing.co.uk%7C8cfd49d71c3d40cc4d7708d870ddbbbc%7C86819931fb92466f8ad749b48fb5a50d%7C0%7C0%7C637383444922347086%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=0SnSvhLnvsD9WBy0HWyqB9T1L1PSM2z9kt7LqsxDag4%3D&reserved=0>

**Comment by jema28** [comment]
Hi @Katie-Hyde, I can't access this unfortunately as it says I need to log in. Could you share screenshots?

**Comment by Katie-Hyde** [comment]
Hi @jema28 you can just google Octopus energy no account needed if not i can screen share Monday

**Comment by jema28** [comment]
Hi @maggiehoughton @Katie-Hyde @thejoefriel 

I hope you're all well. 

I've added the notes from the landscaping session and screenshots from the figma jam file. Here's a quick loom explainer: https://www.loom.com/share/a11c3057952348b09d901a3f0b3ed3f0. 

And here's the notion for quick reference: https://www.notion.so/Landscaping-51fc0fe20b5e435bb0754bcc0477c65e.

Thanks!

**Comment by jema28** [comment]
Hi @Katie-Hyde I've looked at the site but can't see the account views. If you could that would be great, thank you!

**Comment by maggiehoughton** [comment]
Thanks for the video and notes Jem, hope you have a great weekend too!

**Comment by Katie-Hyde** [comment]
Hi @jema28 looks great on the notes and video - actions at bottom i think we were discussing if a user skipped a step the previous would auto turn green as if they had completed - i know I'm eligible - but this would also need a way to clear in case done in error or even if green for completed they could still go into

**Comment by jema28** [comment]
Hi Katie, I'm sorry for the late reply! This slipped through my notifications. I've added this to the notes and going to work this in today
